src/core/TestRunnerConfig.ts

```diff
@@ -8,6 +8,7 @@ export interface TestRunnerConfig {
   server: Server;
   address: string;
   port: number;
+  html?: string;
```

Comment:
```suggestion
  testRunnerHtml?: string;
```

Or something else less generic?